cabal-testsuite/PackageTests/NewConfigure/cabal.test.hs

```diff
@@ -0,0 +1,5 @@
+import Test.Cabal.Prelude as P
+import qualified Test.Cabal.Prelude as P
+
+main = cabalTest . recordMode DoNotRecord $
```

Comment:
DoNotRecord is bad right? You want to record, and then have an out file showing that when you new-configure, some output is put to stdout.